src/frontend/utility/footer/index.js

```diff
@@ -0,0 +1,39 @@
+import './footer.css';
+
+export function Footer({ fixed }) {
+  return (
+    <footer
+      className={`footer--ctr flex-ct-ct flex-vertical sm-s ${
+        fixed && 'fixed'
+      }`}
+    >
+      <span className='md bl mg-half'>Connect With Me</span>
+      <div>
+        <a href='https://twitter.com/home'>
```

Comment:
remove a tag from your code

Reply:
yes they will be replaced with Link component